examples/lammps/simple_lammps/lammps_inputs/lj_bulk.input

```diff
@@ -0,0 +1,31 @@
+units real
+atom_style full
+
+dimension 3
+boundary p p p
+
+special_bonds lj 0.0 0.0 0.5 coul 0.0 0.0 0.5
+
+read_data /Users/mrshirts/work/papers/SCALEMS/scale-ms/examples/lammps/simple_lammps/lammps_inputs/lj_bulk.lmp
```

Comment:
Can we put this in the `testdata` repository and then use a relative path to the git submodule? Better yet, can we write this line while preparing input?